coordinator-server/event-client/actionSniffer.js

```diff
@@ -132,10 +134,15 @@ class ActionSniffer {
     parseEventKey(eventKey) {
         var key = parseInt(eventKey, 0);
         if (AllowedEvents[key] != undefined) {
+            let isDebounced = false;
+            if (DebounceEvents[key] != undefined){
+                isDebounced = true;
+            }
+            
             return {
                 eventName: AllowedEvents[key],
                 capture: true,
-                isDebounced: true, // TODO - make it configurable for each event
+                isDebounced
```

Comment:
Maybe this can be added here on a single line?
`isDebounced: DebounceEvents[key] !== undefined`

Reply:
Added it